include/time_series/internal/specialized_classes.hpp

```diff
@@ -158,6 +158,10 @@ class Vector<SingleProcess, T>
     {
         t = v_[index];
     }
+    std::string get_serialized(int index)
+    {
+      throw std::logic_error("function not implemented for non multiprocess time series");      
```

Comment:
Just an idea, not sure, if this is easily possible:  Could this be implemented as a compile-time error instead of an exception?

Reply:
Agreed. I will investigate how this could be done.

Reply:
I need to merge this to go forward. I will create an issue and get back to this.